DragAndDrop/dependencies.gradle

```diff
@@ -14,19 +14,19 @@ ext {
 
     // DragAndDrop library version code:
     // If you want to publish a new version, bump in one (1) the specific line(s)
-    dragAndDropVersionCode = 04
+    dragAndDropVersionCode = 05
```

Comment:
This seems to be the only place where we add a zero in the front of the digit. This is an Integer, so with or without the zero it represents the same number, right? Would it be better to remove it for consistency with the other version?

Reply:
Great catch, yes I will update that!